public/components/navigation/navigation_test.js

```diff
@@ -0,0 +1,37 @@
+import $ from 'jquery';
+import QUnit from 'steal-qunit';
+import F from 'funcunit';
+
+F.attach(QUnit);
+
+QUnit.module('Navigation Layout', {
+    beforeEach: function () {
+        $('#qunit-fixture').html(
+            '<iframe id="navigation-iframe" src="navigation.html" />');
```

Comment:
Attempt to use the iframe for the sole purpose of the media query manipulation. Reuse everything else from the test window. 

```
<iframe><bitballs-navigation></bitballs-navigation><iframe>
```